README.md

```diff
@@ -29,6 +29,31 @@ Some limitations exist by design:
 Note: The packages `pip`, `setuptools`, `wheel` and `pipx` will be installed/upgraded 
 using the `python` executable from the path.
 
+# Default usage
+
+The default usage requires a project name and a token:
+
+      - name: Publish to pypi
+        uses: coveooss/pypi-publish-with-poetry@v1.0.0
+        with:
+          project-name: my-project-name
+          pypi-token: ${{ secrets.PYPI_TOKEN }}
+
+The project will be published, your changeset will be tagged with a `v1.2.3` kind of tag and revisions will increase automatically.
+
+## About the action's version
+
+In this repository, no tags are mutable by design.
+In other words, we don't keep a `@v1` or `latest` tag that we update as new revisions are released.
+
+Instead, we leverage our own repository tagging feature to provide users with a `v{major}.{minor}.{revision}` tag.
+This is similar to specifying the changeset's sha, but offers more visibility on the evolution of the action
+and allows us to transparently issue security and emergency hotfixes if that ever becomes a necessity.

```

Comment:
> and allows us to transparently issue security and emergency hotfixes if that ever becomes a necessity.

So tags _are_ mutable then. :slightly_smiling_face:

Reply:
removed that part